events/hfe_events_maintenance.txt

```diff
@@ -622,12 +626,16 @@ event = {
 				limit = {
 					is_country_type = primitive
 					any_owned_planet = {
-						has_planet_flag = hfe_varelse_colony
+						OR = {
+							has_planet_flag = hfe_varelse_colony
+							has_planet_flag = colony_beta
+						}
 					}
 				}
 				random_owned_planet = {
 					limit = {
 						has_planet_flag = hfe_varelse_colony
+						has_planet_flag = colony_beta
```

Comment:
No prefix
Also why you need second flag

Reply:
Okay, done